src/raftstore/store/bootstrap.rs

```diff
@@ -53,46 +53,47 @@ pub fn bootstrap_store(engine: &DB, cluster_id: u64, store_id: u64) -> Result<()
     ident.set_cluster_id(cluster_id);
     ident.set_store_id(store_id);
 
-    engine.put_msg(&ident_key, &ident)
+    engines.kv_engine.put_msg(&ident_key, &ident)
 }
 
 // Write first region meta and prepare state.
-pub fn write_prepare_bootstrap(engine: &DB, region: &metapb::Region) -> Result<()> {
+pub fn write_prepare_bootstrap(engines: &Engines, region: &metapb::Region) -> Result<()> {
     let mut state = RegionLocalState::new();
     state.set_region(region.clone());
 
     let wb = WriteBatch::new();
+    let raft_wb = WriteBatch::new();
     try!(wb.put_msg(&keys::region_state_key(region.get_id()), &state));
-    try!(write_initial_state(engine, &wb, region.get_id()));
+    try!(write_initial_state(&wb, &raft_wb, region.get_id()));
     try!(wb.put_msg(&keys::prepare_bootstrap_key(), region));
-    try!(engine.write(wb));
+    try!(engines.kv_engine.write(wb));
+    try!(engines.raft_engine.write(raft_wb));
```

Comment:
then cluster will not bootstrapped, we will have store_id with clster not bootstrapped?    same condition as before,  need remove tikv data https://github.com/pingcap/tikv/blob/master/src/server/node.rs#L135